pom.xml

```diff
@@ -440,6 +441,51 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>jira</artifactId>
+            <version>2.3</version>
+            <exclusions>
```

Comment:
@vivek JIRA client 3.0.0 actually included `org.apache.httpcomponents` classes inside of it which caused a conflict with the jar from the same package. 4.0.0 does not have this problem. 

There is a fix upstream as part of https://github.com/jenkinsci/jira-plugin/pull/126